modules/library/render/src/test/java/org/geotools/renderer/lite/LabelDisplacementModeTest.java

```diff
@@ -372,6 +374,65 @@ public void testDisplacementVerticalBothConflictDisabledMultiLayer() throws Exce
         ImageAssert.assertEquals(new File(refPath), image, 900);
     }
 
+    // https://osgeo-org.atlassian.net/browse/GEOS-8975
+    // https://osgeo-org.atlassian.net/projects/GEOT/issues/GEOT-6253
+    public void testGEOT6253() throws Exception {
```

Comment:
The pointers to the ticket are fine, but give the test a human readable name. Same goes for the sld style file name and feature type name.